criu/util.c

```diff
@@ -30,6 +30,8 @@
 
 #include "linux/mount.h"
 
+#include "linux/mount.h"
+
```

Comment:
Duplicate include statement. 4997a096e ("util: introduce the mount_detached_fs helper") Is already there.